### ISSUE TITLE
A few Mathematical errors and typos

### ISSUE THREAD

**Issue by danuSHekar** [cs231n/cs231n.github.io#234]
Under Module 1 of the "Optimisation: Stochastic Gradient Descent" chapter, there are multiple instances where it is mentioned that W has 30730 parameters. But according to the previous chapter and the fact that they are 32x32 pixel images in RGB scheme to be classified in 10 classes (which means 32*32*3*10=30720) it should be 30720 instead of 30730. There are multiple instances where this mistake is made. Please look into them.
Under section "Mini-batch gradient descent" of the Optimisation chapter in Module 1, there is an incorrect sentence: "the training data can have on order of millions of examples" instead of "the training data can have an order of millions of examples".

**Comment by kevinzakka** [comment]
The calculation is implicitly including the bias term, so the weight matrix has the shape `(10, 3073)`. Refer to the bias trick section [here](https://cs231n.github.io/linear-classify/).

**Comment by danuSHekar** [comment]
Ok, that makes sense. My bad.

**Comment by danuSHekar** [comment]
I hope you haven't missed the comment on the typo.
